### PR TITLE
References

### PR DESCRIPTION
**Pull request by liatsade**
https://www.gitbook.com/book/amitaibu/the-gizra-way/edit#/edit/References/references.md

@amitaibu 
During my learning I found some good training materials, I think we should have them in GitBook.

### CODE REVIEW

File: references.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# References
+
+## Tools
+
+| Topic | Link |
+| -- | -- |
+| PHP for beginners | http://www.sololearn.com/Play/PHP# |
+| Git - try git | https://try.github.io/levels/1/challenges/1 |
```
Comment (amitaibu) [inline references.md]: Topic can be have the link for example `[PHP for beginners](http://www.sololearn.com/Play/PHP)`

Comment (amitaibu) [inline references.md]: Also lets divide this into sections . PHP/ CSS/ JS etc;

**Comment by liatsade** [comment]
@RachelBaram please do english review.

**Comment by RachelBaram** [comment]
@liatsade Proofread.

**Comment by amitaibu** [comment]
I'm not sure about this, I think it shouldn't be merged. Every person comes from different background, so a general references sounds wrong. I think it's worth adding those references per chappter, as "further reading".

Please open if you think differently.

**Comment by liatsade** [comment]
Well, the question is if we are going to write chapter on every subject...
For example Git is very important that every employee will know, because it is main tool in our work, but is it necessary to teach Git or to write post about it? or just reference to a good tutorial so everyone can learn it by himself...
I can ask the same about more subject in that list.

**Pull request closed without merging**